content/guides/output-management.md

```diff
@@ -3,85 +3,262 @@ title: Output Management
+
+``` bash
+Hash: 5f3b0265b87c603b4a0f
+Version: webpack 2.6.1
+Time: 539ms
+           Asset     Size  Chunks                    Chunk Names
+vendor.bundle.js   544 kB       0  [emitted]  [big]  vendor
+   app.bundle.js  2.81 kB       1  [emitted]         app
+   [0] ./~/lodash/lodash.js 540 kB {0} [built]
+   [1] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [2] (webpack)/buildin/module.js 517 bytes {0} [built]
+   [3] ./src/index.js 172 bytes {1} [built]
+   [4] multi lodash 28 bytes {0} [built]
+```
+
+We can see that webpack generates our `vendor.bundle.js` and `app.bundle.js` files, which we also specified in our `index.html` file. But what would happen if we changed the name of one of our entry points? The generated bundles would be renamed on a build, but our `index.html` file would still reference the old names. Let's fix that with the [`HtmlWebpackPlugin`](/plugins/html-webpack-plugin).
+
+
+## Setting up HtmlWebpackPlugin ##
+
+First install the plugin and adjust the `webpack.config.js` file:
+
+``` bash
+npm install --save-dev html-webpack-plugin
+```
+
+__webpack.config.js__
+
+``` diff
+  const path = require('path');
++ const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+  module.exports = {
+    entry: {
+      app: './src/index.js',
+      vendor: ['lodash']
+    },
++   plugins: [
++     new HtmlWebpackPlugin()
++   ],
+    output: {
+      filename: '[name].bundle.js',
+      path: path.resolve(__dirname, 'dist')
+    }
+  };
+```
 
+Now run `npm run build` and you should see something similar to this:
+
+``` bash
+Hash: 81f82697c19b5f49aebd
+Version: webpack 2.6.1
+Time: 854ms
+           Asset       Size  Chunks                    Chunk Names
+vendor.bundle.js     544 kB       0  [emitted]  [big]  vendor
+   app.bundle.js    2.81 kB       1  [emitted]         app
+      index.html  249 bytes          [emitted]
+   [0] ./~/lodash/lodash.js 540 kB {0} [built]
+   [1] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [2] (webpack)/buildin/module.js 517 bytes {0} [built]
+   [3] ./src/index.js 172 bytes {1} [built]
+   [4] multi lodash 28 bytes {0} [built]
+Child html-webpack-plugin for "index.html":
+       [0] ./~/lodash/lodash.js 540 kB {0} [built]
+       [1] ./~/html-webpack-plugin/lib/loader.js!./~/html-webpack-plugin/default_index.ejs 538 bytes {0} [built]
+       [2] (webpack)/buildin/global.js 509 bytes {0} [built]
+       [3] (webpack)/buildin/module.js 517 bytes {0} [built]
+```
+
+If you open `index.html` in your code editor, you'll see that the `HtmlWebpackPlugin` has created an entirely new file for you and that all the bundles are automatically added. This is great, but it also has overwritten everything else we had in our file.
+
+In most cases you probably want to provide a certain template to the `HtmlWebpackPlugin`, so that you can still have your own `index.html` file, but have webpack automatically add generated files.
+
+
+## Adding a template to HtmlWebpackPlugin ##
```

Comment:
I would drop this as an entire section and, if anything, just mention that you can pass a custom template if necessary, maybe linking to that . It's also probably worth linking to the popular [html-webpack-template package](https://github.com/jaketrent/html-webpack-template) as you can get pretty far with that. 

I have multiple projects that use this plugin and haven't needed to write a custom template -- plus the things you added with the custom template are available from the base plugin (and almost anything else you'd need is available from `html-webpack-template`). I know this might sound nitpicky but I think we should really try to avoid replicating documentation and instead focus on _just introducing_ a variety of approaches and then let the user take their own path from there.